lib/utils/tests-options-validator.js

```diff
@@ -79,13 +121,13 @@ function validatePartitionSplit(partitions, split) {
  * Ensures that there is no partition listed twice
  *
  * @private
- * @param {Array} partitions
+ * @param {Array} arr
  */
-function validatePartitionsUnique(partitions) {
-  partitions = partitions.sort();
-  for (var i = 0; i < partitions.length - 1; i++) {
-    if (partitions[i] === partitions[i + 1]) {
-      throw new Error('You cannot specify the same partition twice.');
+function validateElementsUnique(arr) {
+  arr = arr.sort();
+  for (var i = 0; i < arr.length - 1; i++) {
+    if (arr[i] === arr[i + 1]) {
+      throw new Error('You cannot specify the same value twice.');
```

Comment:
The error message really should indicate what type of value it's talking about (e.g. partition ID). The message as given will just confuse the user (this is like some of those log messages we see that we have no idea where they are coming from).

Reply:
also, how about we also display the values in the error, so the user will see exactly which values are repeated.